core/server/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -59,7 +59,7 @@
   public static final String GET_VERSION = "version";
   public static final String GET_METRICS = "metrics";
 
-  private final AlluxioWorker mWorker = AlluxioWorker.get();
+  private final AlluxioWorker mWorker = DefaultAlluxioWorker.get();
```

Comment:
how about instead of calling `DefaultAlluxioWorker.get()`,  make a factory method in `AlluxioWorker` which by default returns `DefaultAlluxioWorker`?